webserver/templates/data/summary.html

```diff
@@ -98,13 +98,13 @@ <h3 class="text-center">
     <ul>
       <li>
         <a href="{{ url_for('data.view_low_level', mbid=metadata['mbid'], n=offset) }}">Low-level</a>
-        (<a href="{{ url_for('api.get_low_level', mbid=metadata['mbid'], n=offset) }}">raw JSON</a>)
+        (<a href="{{ url_for('api_v1_core.get_low_level', mbid=metadata['mbid'], n=offset) }}">raw JSON</a>)
       </li>
       {% if data.highlevel %}
         <li>
           {# FIXME(roman): Offset might not point to the right high-level data there. #}
```

Comment:
I think this is fixed now with the changes I made to the hl load with the database changes: https://github.com/metabrainz/acousticbrainz-server/blob/master/db/data.py#L334